### ISSUE TITLE
Social network aspect

### ISSUE THREAD

**Issue by clancykelly** [voxverus/JobTalk#3]
Ideally users will be able to message and connect with other users to network, share information, etc. For starters this can entail a simple profile page presenting whatever work information users want to make public, and a messaging system.

**Comment by WW-Web-Infra** [comment]
@clancykelly if you plan to keep it anonymous, ow would you recommend for the profile to be managed? what kind of details a profile must have?

**Comment by clancykelly** [comment]
The submitted workplace reviews should be anonymous, but users (if they choose) could have a kind of profile page (university affiliation/position/experience/etc) so users at different institutions/countries could connect/seek advice/etc. We're currently working on the exact content of the reviews/profiles and we can upload a draft here soon!

**Comment by WW-Web-Infra** [comment]
@clancykelly That's a great idea!! So What is the plan for the project? Time schedule, technology stack, etc!

**Comment by clancykelly** [comment]
So! We hope we can get a skeleton of the site--like very minimum the reviews portion together by/during MozSprint in June. That should be pretty simple--effectively just a form submission. If we can get some simple networking stuff together by then too, that would be grand. As for the tech stack, lots of things would work so it's sort of up to what our majority of contributors are comfortable/familiar with--if you have suggestions, we'd love to hear em!

**Comment by WW-Web-Infra** [comment]
@clancykelly Sorry for the delay! I would be glad to explore! So for the front end as well as back-end what would you recommend ? I think a good UI (review page) and simple profile  would be great till June!

**Comment by aromartins** [comment]
Yes! that is the goal: to have a simple profile and review page setup till June ideally ! Now working on making the flow very clear and visual so we can easily share our vision with collaborators that can help us achieve that goal!

**Comment by clancykelly** [comment]
Yeah! So I confess my knowledge is some years out of date but I believe django, flask, python, ruby would all work well, for front end angular/ react or even simpler, it needn't be fancy.

**Comment by WW-Web-Infra** [comment]
@aromartins @clancykelly Thank you for the response. I will research a bit and will let you know :) For front end, I am aware of JS but struggling for back end.

**Comment by clancykelly** [comment]
Awesome, thank you!!

**Comment by WW-Web-Infra** [comment]
@clancykelly regret to say that, I may not be able to contribute due to other commitments. I hope that soon I can get in touch with you :) Thanks :)

**Comment by clancykelly** [comment]
Zero worries! Thanks so much for your interest and input! Keep in touch!

**Comment by EmmanuelBeja** [comment]
am interested. Will work on a php mysql sketch

**Comment by clancykelly** [comment]
Hey! Awesome, thanks! Want to join the slack team to join in the general discussion?  https://jobtalk-self-sign-in.herokuapp.com/ We have some folks starting on the reviews portion of the site, mostly in django. Asking folks to add to this to do list so we know who's working exactly on what, so if you want to work on this want to just post a little blurb about what you're planning to do? Thanks!
 https://github.com/voxverus/JobTalk/blob/master/ToDo.md

**Comment by aromartins** [comment]
Hi Emmanuel - I notice you have not joined our slack group, but if you still wan to do some work with us, we have  updated a TO DO list here and you can also note down any relevant pieces of info on what you might be working on. Thanks again for your interest! https://docs.google.com/spreadsheets/d/1KtLGTf_zE2kT-rax588XQgFOS95T3cLweqSn-H5Bx3I/edit?usp=sharing
